### PR TITLE
Remove xamlToolsPreview switch

### PR DESCRIPTION
**Pull request by mgoertz-msft**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1719,12 +1719,6 @@
             "default": null,
             "description": "%configuration.dotnet.server.crashDumpPath%"
           },
-          "dotnet.enableXamlToolsPreview": {
```
Comment (dibarbet) [inline package.json:1722]: Lets keep the option, but change the default

**Review by dibarbet — CHANGES_REQUESTED** [review]

**Comment by dibarbet** [comment]
Also, this needs to target the prerelease branch if its going into release next week.

File: package.json (hunk 1 above)
Comment (mgoertz-msft) [reply package.json:1722]: The default is already `true`. The idea (and plan) is to remove the option altogether for release.

**Comment by mgoertz-msft** [comment]
Ugh, I guess it isn't as simple as changing the target branch to prerelease

**Pull request closed without merging**